nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_depth.py

```diff
@@ -93,6 +99,27 @@ def depth_indicator(self, depth_indicator: int) -> None:
         self._depth_indicator = depth_indicator
         self._is_search_space_obsolete = True
 
+    def load_state(self, state: Dict[str, Any]) -> None:
+        """
+        Initializes object from the state.
+        :param state: Output of `get_state()` method.
+        """
+        active_config = state[self._state_names.ACTIVE_CONFIG]
+        self.activate_subnet_for_config(active_config)
+        self.depth_indicator = state[self._state_names.DEPTH_INDICATOR]
+
+    def get_state(self) -> Dict[str, Any]:
+        """
+        Returns a dictionary with Python data structures (dict, list, tuple, str, int, float, True, False, None) that
+        represents state of the object.
+        :return: state of the object
+        """
+        active_config = self.get_active_config()
```

Comment:
```
state = super().get_state()
state[self._state_names.DEPTH_INDICATOR] = self.depth_indicator
```

Reply:
Thanks! Please review the new design and suggest changes for elastic dimensions that don't have an elastic indicator, e.g., kernel. Thanks!